code/espurna/homeassistant.ino

```diff
@@ -317,6 +349,7 @@ void haSetup() {
 
     // Main callbacks
     espurnaRegisterReload(_haConfigure);
+    espurnaRegisterLoop(_haLoop);
```

Comment:
Is it really worth it to have it being checked in the main loop all the time? Wouldn't it be better t use a ticker?

Reply:
One option is to make additional API for such one-off functions to not overuse Ticker. Somehing accepting std::function with bound args.
Also see #1370

And I think I should also check if client is connected before starting the process.